build/v1/types.go

```diff
@@ -977,7 +977,11 @@ type BuildConfigStatus struct {
 	LastVersion int64 `json:"lastVersion" protobuf:"varint,1,opt,name=lastVersion"`
 
 	// ImageChangeTriggers is used to capture the runtime state of any ImageChangeTrigger specified in the BuildConfigSpec,
```

Comment:
// ImageChangeTriggers captures the runtime state of any ImageChangeTrigger specified in the BuildConfigSpec